src/main/java/com/github/platymemo/alaskanativecraft/item/AlaskaItems.java

```diff
@@ -39,7 +39,10 @@ public class AlaskaItems {
     public static final Item CLOUDBERRIES;
     public static final Item RASPBERRIES;
     public static final Item SALMONBERRIES;
+    public static final Item LABRADOR_TEA_LEAVES;
+    public static final Item LABRADOR_TEA_SEEDS;
     public static final SuspiciousStewItem AKUTAQ;
+    public static final LabradorTeaItem LABRADOR_TEA;
```

Comment:
Personally, I feel the term tundra tea is better: that is what it's colloquially known as. It's more representative of the cultures it is used in.

Reply:
Also works better in MC